protocols/08aeaa/README.md

```diff
@@ -0,0 +1,59 @@
+# Normalization
```

Comment:
Can we give this a more detailed title? 'Normalization' is pretty broad and this protocol is doing something a bit more complex than a normalization (normalization and transfer).

Maybe `Custom Normalization & Transfer` or something like that...

Reply:
Agreed! I totally overlooked the titles as I was using another README as my base template.